### PR TITLE
doc: update directories/files for new prop process

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: proposals/approved/EXPECTATIONS/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The bootstrap committee to finalize for consideration of the Boards and Legal.
 
 The needs of the projects and ecosystem stakeholders being voiced through the governance of the foundation for which they are represented is vital for the foundation to succeed.
 
-## What is neccessary to get this proposal to stage 3
+## What is neccessary to get this proposal to approved
```
Comment (eemeli) [inline proposals/approved/EXPECTATIONS/README.md:24]:
```suggestion
## What is necessary to approve this proposal
```

File: proposals/approved/GOVERNANCE/README.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ The bootstrap committee
 
 We need basic governance to operate
 
-## What is neccessary to get this proposal to stage 3
+## What is neccessary to get this proposal to approved
```
Comment (eemeli) [inline proposals/approved/GOVERNANCE/README.md:24]:
```suggestion
## What is necessary to approve this proposal
```

**Review by eemeli — APPROVED** [review]

File: proposals/README.md [hunk 3]
```diff
@@ -2,31 +2,26 @@
...
-* [Community Board representation](./stage-3/COMMUNITY_BOARD_REPRESENTATION)
-* [CPC Charter](./stage-3/CPC_CHARTER)
-* [Expectations](./stage-3/EXPECTATIONS)
-* [Governance](./stage-3/GOVERNANCE)
-* [Individual Membership program](./stage-3/INDIVIDUAL_MEMBERSHIP)
-* [Project Onboarding](./stage-3/PROJECT_ONBOARDING)
-* [Project Progression](./stage-3/PROJECT_PROGRESSION)
-* [Travel Fund](./stage-3/TRAVEL_FUND)
-* [Code of Conduct approach](./stage-3/CODE_OF_CONDUCT)
+## incubating
```
Comment (eemeli) [inline proposals/README.md:5]:
```suggestion
## Incubating 
```

File: proposals/README.md [hunk 4]
```diff
@@ -2,31 +2,26 @@
...
+* [Regular Town Halls](./incubating/REGULAR_TOWN_HALLS)
+* [Shared Interfaces](./incubating/SHARED_INTERFACES)
+* [Standards Outreach](./incubating/STANDARDS_OUTREACH)
+* [Charter Review](./incubating/CHARTER_REVIEW)
+* [Collab Summit](./incubating/COLLAB_SUMMIT)
+* [Individual Supporter Program](./incubating/SUPPORTER_PROGRAM)
+* [Collaboration Network](./incubating/COLLABORATION_NETWORK)
+* [Security Reporting](./incubating/SECURITY_REPORTING)
+
+## approved
```
Comment (eemeli) [inline proposals/README.md:17]:
```suggestion
## Approved
```

**Review by joesepi — APPROVED** [review]

**Comment by mhdawson** [comment]
Accepted suggestion.

**Pull request merged**